lib/siwapp_web.ex

```diff
@@ -91,6 +91,8 @@ defmodule SiwappWeb do
       import SiwappWeb.ErrorHelpers
       import SiwappWeb.Gettext
       alias SiwappWeb.Router.Helpers, as: Routes
+
+      alias SiwappWeb.GraphicHelpers, as: Graphics
```

Comment:
I think this is not the right place for this. Probably in the views would be better. 
This is the APP module, is just too big for this

Reply:
Ok, I understand. I'll move it